### PR TITLE
Real data

### PR DESCRIPTION
**Pull request by alfredbirk**

### CODE REVIEW

File: webapp/src/containers/MapContainer.tsx [hunk 1]
```diff
@@ -238,6 +278,8 @@ const MapContainer: FunctionComponent<MapContainerProps> = ({ match, history })
           </Row>
           <Row>
             <Mapbox
+              gknr={currentBasicStatisticalUnitNumber}
```
Comment (mariusaarsnes) [inline webapp/src/containers/MapContainer.tsx]: you don't need this. use elemsState.unitNumber. And we aren't using 'gknr' anymore :)

File: webapp/src/containers/MapContainer.tsx [hunk 2]
```diff
@@ -101,21 +102,59 @@ interface MapContainerProps {
   currentTractName: String | null;
   history: any;
   updated: String;
+  dates: Date[];
 }
 
 const MapContainer: FunctionComponent<MapContainerProps> = ({ match, history }) => {
+  const [currentBasicStatisticalUnitNumber, setcurrentBasicStatisticalUnitNumber] = useState(null);
```
Comment (mariusaarsnes) [inline webapp/src/containers/MapContainer.tsx]: there is no need for this value. We already have it in elemsState

File: webapp/src/components/Mapbox/Mapbox.jsx [hunk 3]
```diff
@@ -240,39 +293,112 @@ const Mapbox = ({ tractNumber, unitNumber, handleSelect, timestamp }) => {
...
-          closeButton={false}
-        >
-          <div style={{ color: '#14598B', fontSize: '14px' }}>
-            {currentHoverInfo.hoverElem.grunnkretsnavn
-              ? currentHoverInfo.hoverElem.grunnkretsnavn
-              : currentHoverInfo.hoverElem.DELOMRNAVN}
-          </div>
-        </Popup>
-      );
+  getMap = () => {
```
Comment (mariusaarsnes) [inline webapp/src/components/Mapbox/Mapbox.jsx]: i see all this is commented out. do we need it for anything?

File: webapp/src/containers/MapContainer.tsx [hunk 4]
```diff
@@ -101,21 +102,59 @@ interface MapContainerProps {
...
   };
   const [elemsState, elemsDispatch] = useReducer(elemsReducer, initialElemsState);
-  const [dates, setDates] = useState([new Date(2019, 5, 20), new Date(2019, 5, 25)]);
   const [modalVisible, setModalVisible] = useState(false);
-  const [updated, setUpdated] = useState(mockdata[0].timestamp);
+  const [updated, setUpdated] = useState(dates[0]);
+  const [dataset, setDataset] = useState([]);
 
   useEffect(() => {
+    const { gknr } = match.params;
```
Comment (mariusaarsnes) [inline webapp/src/containers/MapContainer.tsx]: Should be rewritten to use elemsState instead. using ucrrentBasicStatisticalUnitNumber is redundant

**Comment by mariusaarsnes** [comment]
Closes #98

File: webapp/src/containers/MapContainer.tsx (hunk 4 above)
Comment (mariusaarsnes) [reply webapp/src/containers/MapContainer.tsx]: Will 'gknr' ever not be undefined? If I have understood the code correctly you should use unitNumber instead. 

When destructuring like this I believe the name of the variable must also be present in match.params

File: webapp/src/components/MobilityTable/MobilityTable.js [hunk 5]
```diff
@@ -69,15 +68,16 @@ const useStyles = makeStyles(theme => ({
   }
 }));
 
-const MobilityTable = ({ timestamp, handleSelect, tractNumber }) => {
+const MobilityTable = ({ handleSelect, tractNumber, dataset }) => {
+  // console.log(dataset);
```
Comment (emilps) [inline webapp/src/components/MobilityTable/MobilityTable.js]: Reason for this to be in the code?

File: webapp/src/containers/MapContainer.tsx [hunk 6]
```diff
@@ -101,19 +102,123 @@ interface MapContainerProps {
...
+          
+        }
+      })
+      .catch(error => {
+        console.log(error);
+      });
+  };
+
+  const getAllData = () => {
+    console.log("Retrieving data...");
```
Comment (emilps) [inline webapp/src/containers/MapContainer.tsx]: Remove console.log

File: webapp/src/containers/MapContainer.tsx [hunk 7]
```diff
@@ -101,19 +102,123 @@ interface MapContainerProps {
...
+            host: 'localhost',
+            port: 5000
+          }
+        }
+      )
+      .then(response => {
+        if (offset < response.data.count) {
+          setDataPoint(response.data.datapoints);
+          getDataRecursive(offset + response.data.limit);
+          // console.log("offset:", offset + response.data.limit);
```
Comment (emilps) [inline webapp/src/containers/MapContainer.tsx]: Need this?

File: webapp/src/containers/MapContainer.tsx [hunk 8]
```diff
@@ -101,19 +102,123 @@ interface MapContainerProps {
...
+        }
+      )
+      .then(response => {
+        if (offset < response.data.count) {
+          const dataPoint = response.data.datapoints;
+          dataPoint.forEach(element => {
+            newDataSet.push(element);
+          });
+          setDataSet(newDataSet)
+          console.log("New dataset length:", newDataSet.length);
```
Comment (emilps) [inline webapp/src/containers/MapContainer.tsx]: Remove console.log

File: webapp/src/containers/MapContainer.tsx (hunk 6 above)
Comment (alfredbirk) [reply webapp/src/containers/MapContainer.tsx]: This console.log will be needed for future testing.

File: webapp/src/containers/MapContainer.tsx (hunk 8 above)
Comment (alfredbirk) [reply webapp/src/containers/MapContainer.tsx]: This console.log will be needed for future testing.

**Pull request merged**